server/src/main/java/com/scalar/db/server/AdminService.java

```diff
@@ -28,12 +29,17 @@ public AdminService(GateKeeper gateKeeper) {
   @Override
   public void pause(PauseRequest request, StreamObserver<Empty> responseObserver) {
     gateKeeper.close();
+    long maxPauseWaitTime =
+        request.getMaxPauseWaitTime() != 0
```

Comment:
if `max_pause_wait_time` is not set, getMaxPauseWaitTime will return `0`.

reference:
https://developers.google.com/protocol-buffers/docs/javatutorial

```
a system default is used: zero for numeric types, the empty string for strings, false for bools. 
```